lib/temporal/client.rb

```diff
@@ -9,15 +9,50 @@
 
 module Temporal
   class Client
+    # @return [String] Namespace used for client calls.
     attr_reader :namespace
 
-    # TODO: More argument to follow for converters, codecs, etc
+    # Create a Temporal client from a service client.
+    #
+    # @param connection [Temporal::Connection] A previously established connection with the
```

Comment:
Are you having to manually keep the types in the doc strings in sync w/ the types in your `rbs` files? Is there some kind of check that we can enable in CI to ensure the sync is accurate?

Reply:
Unfortunately yes, it's manual. There's a plugin to generate RBS from YARD docs, but not the other way round. I don't think there's a validator, but we can probably write one if needed later.

Alternatively we can remove all the types from docs… but that's probably not the best idea from the user's convenience point of view.